### PR TITLE
[Workers] Add info in required permissions for mTLS cert upload

### PR DESCRIPTION
**Pull request by markdembo**
Add warning for required permissions

### CODE REVIEW

File: content/workers/runtime-apis/mTLS.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ Currently, mTLS for Workers is not supported for domains [proxied](/dns/manage-d
 
 First, upload a certificate and its private key to your account using the [`wrangler mtls-certificate`](/workers/wrangler/commands/#mtls-certificate) command:
 
+{{<Aside type="warning">}}
+
+This command requires "Account: SSL and Certificates" write permissions.
```
Comment (deadlypants1973) [inline content/workers/runtime-apis/mTLS.md]:
```suggestion
The `wrangler mtls-certificate upload` command requires Account: SSL and Certificates write permissions.
```
Are there instructions for how to establish this "Account: SSL and Certificates write permissions"? If something is necessary, there should be followup guidance on how to accomplish it.

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]
PCX

File: content/workers/runtime-apis/mTLS.md (hunk 1 above)
Comment (markdembo) [reply content/workers/runtime-apis/mTLS.md]: Added further instructions.

**Comment by samirbani** [comment]
في الأربعاء، ١٩ يوليو ٢٠٢٣ ١٠:٠٨ ص Mark Dembo ***@***.***>
كتب:

File: content/workers/runtime-apis/mTLS.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ First, upload a certificate and its private key to your account using the [`wran
 
 {{<Aside type="warning">}}
 
-This command requires "Account: SSL and Certificates" write permissions.
 
+The `wrangler mtls-certificate upload` command requires ["SSL and Certificates Edit"](https://developers.cloudflare.com/fundamentals/api/reference/permissions/) API token scope. If your are using the OAuth flow triggered by  `wrangler login`, the correct scope is set automatically. If you are using API tokens, follow the guide ["Create an API token"](https://developers.cloudflare.com/fundamentals/api/get-started/create-token/) accordingly to se the right scope.
```
Comment (deadlypants1973) [inline content/workers/runtime-apis/mTLS.md]:
```suggestion
The `wrangler mtls-certificate upload` command requires the [SSL and Certificates Edit API token scope](/fundamentals/api/reference/permissions/). If you are using the OAuth flow triggered by  `wrangler login`, the correct scope is set automatically. If you are using API tokens, refer to [Create an API token](https://developers.cloudflare.com/fundamentals/api/get-started/create-token/) to set the right scope for your API token.
```

**Review by deadlypants1973 — APPROVED** [review]
PCX approved with suggestion

**Pull request merged**